test-harness/specs/security-scheme-validation-AC-11.only_oas3.txt

```diff
@@ -0,0 +1,29 @@
+====test====
+Resource is returned when a bearer token is provided as a header.
```

Comment:
Assumed: it's OK to send any token to pass the security.
So no checking if the passed token is valid.